tests/scripts/components-configuration-crypto.sh

```diff
@@ -162,6 +162,25 @@ component_test_rsa_no_crt () {
     tests/context-info.sh
 }
 
+component_test_config_no_entropy () {
```

Comment:
Just to make sure: this one doesn't have legacy/PSA variants because PSA doesn't work without entropy, right?

Reply:
Within the context of this PR: there's no PSA variant because the config file doesn't have a line with `//#define MBEDTLS_USE_PSA_CRYPTO`, which is how `test-ref-configs.pl` determined whether to also run a PSA-modified variant of the configuration.

Outside the context of this PR: indeed, the reason `config-no-entropy.h` doesn't have a PSA variant is that PSA can't be configured with an RNG. (You can use an NV seed as the sole “entropy”, but that still goes through the entropy module.)